Lib/fontmake/font_project.py

```diff
@@ -20,11 +20,13 @@
 import logging
 import math
 import os
+import shutil
 import tempfile
 try:
     from plistlib import load as readPlist  # PY3
 except ImportError:
     from plistlib import readPlist  # PY2
+import xml.etree.ElementTree as etree
```

Comment:
For compatiblity with Python 2, I'd suggest here you try to import the faster C implementation `cElementTree` first, and if that fails (on platforms that might not have the `_elementtree` C extension module) use the pure Python one. Python 3 does that automatically when you `import xml.etree.ElementTree`.

```python
try:
	import xml.etree.cElementTree as etree
except ImportError:
	import xml.etree.ElementTree as etree
```